client/src/components/Language/LanguageBar/index.js

```diff
@@ -17,6 +17,21 @@ const props = {
   iconColor: 'neutralMain',
 };
 
+const useBack = (navigate) => {
```

Comment:
you don't need to pass navigate as a parameter here senice it  comes from `useNavigate` hook, you can call it here 
```suggestion
const useBack = () => {
const navigate = useNavigate();
```